api/src/main/java/eu/darkbot/api/game/other/Movable.java

```diff
@@ -29,15 +29,17 @@ public interface Movable extends Entity {
 
     /**
      * Reads in-game angle which is, where entity looks at, not where it flies to.
+     * <br>Value is in the range of <i>pi</i> to <i>2pi</i>
      *
      * @return angle of the {@link Movable} in-game as radians.
      */
     double getAngle();
 
     /**
      * Comparing with {@link Double#MIN_VALUE} means that angle was never updated.
+     * <br>Value is in the range of -<i>pi</i> to <i>pi</i>
      *
-     * @return angle based on current position and destination
+     * @return angle based on current position and destination, is rotated by 50% compared to {@link LocationInfo#getAngle()}
```

Comment:
What does "rotated by 50%" even mean? do you just mean that it is rotated by 180 degrees?